packages/babel-core/test/fixtures/transformation/source-maps/full/source-map.json

```diff
@@ -1,11 +1,15 @@
 {
   "version": 3,
-  "sources": ["source-maps/full/input.js"],
   "names": [
     "arr",
     "map",
     "x"
   ],
-  "mappings": "AAAAA,GAAG,CAACC,GAAJ,CAAQ,UAAAC,CAAC;AAAA,SAAIA,CAAC,GAAGA,CAAR;AAAA,CAAT",
-  "sourcesContent": ["arr.map(x => x * x);"]
-}
+  "sources": [
+    "source-maps/full/input.js"
+  ],
+  "sourcesContent": [
+    "arr.map(x => x * x);"
+  ],
+  "mappings": "AAAAA,GAAG,CAACC,GAAJ,CAAQ,UAAAC,CAAC;EAAA,OAAIA,CAAC,GAAGA,CAAR;AAAA,CAAT"
```

Comment:
The diff here:
```diff
- AAAAA,GAAG,CAACC,GAAJ,CAAQ,UAAAC,CAAC;AAAA,SAAIA,CAAC,GAAGA,CAAR;AAAA,CAAT
+ AAAAA,GAAG,CAACC,GAAJ,CAAQ,UAAAC,CAAC;EAAA,OAAIA,CAAC,GAAGA,CAAR;AAAA,CAAT
```

The changed `AAAA,SAAIA` -> `EAAA,OAAIA` is the two spaces used for indentation. `A` means 0, and `E` means , so we moved the generated column 2 times when inserting the new marking. The `S` -> `O` is because markings are deltas, so we subtracted two columns from `S` gives us `O`.